crates/solidity/testing/perf/npm/src/benchmarks/common.mts

```diff
@@ -0,0 +1,137 @@
+import fs from "node:fs";
+import path from "node:path";
+
+export const hasGC = typeof global.gc == "function";
```

Comment:
nit: for a lot of these exported variables/functions, they are only used locally in this file. I suggest making them private/inline.

Reply:
there are not being used, so I'm removing them